features/poller/shell/src/main/java/org/opennms/netmgt/poller/shell/Test.java

```diff
@@ -140,7 +142,15 @@ public MonitoredService doInTransaction(TransactionStatus status) {
 
         ServiceMonitor monitor = null;
         if (monitorClass == null) {
-            monitor = pollerConfig.getServiceMonitor(serviceName);
+            final Optional<Package.ServiceMatch> service = pkg.findService(serviceName);
+            if (!service.isPresent()) {
+                System.err.printf("Error: Service %s not defined%n", serviceName);
```

Comment:
`%n`  intentional?

Reply:
Yes, It's used every where else in the file, so I just keeped the style conistent.